staging/src/k8s.io/kubectl/pkg/cmd/proxy/proxy.go

```diff
@@ -200,6 +200,10 @@ func (o ProxyOptions) Validate() error {
 func (o ProxyOptions) RunProxy() error {
 	server, err := proxy.NewServer(o.staticDir, o.apiPrefix, o.staticPrefix, o.filter, o.clientConfig, o.keepalive)
 
+	if err != nil {
+		return err
```

Comment:
this looks correct. the other `klog.Fatal(err)` lines in this method should be changed to return the err instead as well